test/helpers/supportDefaultAssignment.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function supportDefaultAssignment() {
+	try {
+		return true;
```

Comment:
Check how the other helpers work:

https://github.com/webpack/webpack/blob/master/test/helpers/supportsDefaultArgs.js
https://github.com/webpack/webpack/blob/master/test/helpers/supportsBlockScoping.js
https://github.com/webpack/webpack/blob/master/test/helpers/supportsES6.js